apis/beacon/attestations/rewards.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  operationId: getAttestationsRewards
+  summary: Get attestations rewards
+  description: Retrieves rewards info for attestations for validators specified in id. If none is specified, return rewards info for every validators.
+  tags:
+    - Beacon
+    - Rewards
```

Comment:
CI is complaining that this needs to be part of the global tags: https://github.com/ethereum/beacon-APIs/actions/runs/3438408607/jobs/5774102666